package.json

```diff
@@ -1,12 +1,16 @@
 {
   "name": "@privacybydesign/irmajs",
-  "version": "0.1.4",
+  "version": "0.1.5",
   "description": "",
   "main": "dist/irma.node.js",
   "scripts": {
     "develop": "webpack --mode development --watch",
     "build": "webpack --mode production"
```

Comment:
I think you just forgot to add a comma before the previous line. Now it is not valid JSON anymore. Maybe you can add this to your commit.